pkg/apis/networking/types.go

```diff
@@ -90,6 +90,25 @@ type NetworkPolicyPort struct {
 	Port *intstr.IntOrString
 }
 
+// CIDR describes a Network CIDR to select
+type CIDR string
+
+// Except is a slice of CIDRs to not include within an IpBlockRule
+type Except []string
+
+// IpBlockRule describes a particular CIDR (Ex. "192.168.1.1/24") that is allowed to the pods
+// matched by a NetworkPolicySpec's podSelector. The except entry describes CIDRs that should
+// not be included within this rule.
+type IpBlockRule struct {
```

Comment:
Capital P. `IPBlockRule`. Acronyms are always either all-caps or all-lowercase. (So `IPBlockRule` here and `ipBlockRule` in the JSON.)